src/tests/ftest/control/daos_admin_privileged.yaml

```
@@ -9,15 +9,12 @@ setup:
   start_servers: False
   start_agents_once: False
   start_servers_once: False
-  agent_manager_class: Orterun
-  server_manager_class: Orterun
 server_config:
   name: daos_server
   port: 10001
   servers:
-    # Uncomment once DAOS-4287 has been closed.
-    # bdev_class: nvme
-    # bdev_list: ["aaaa:aa:aa.a","bbbb:bb:bb.b"]
+    bdev_class: nvme
+    bdev_list: ["0000:81:00.0"]
```

Is there a reason this was this reduced to one NVMe?
```suggestion
    bdev_list: ["aaaa:aa:aa.a","bbbb:bb:bb.b"]
```

---

Since this is a basic test for nvme storage, 1 nvme list would be sufficient, unless it is required, please advise.